Misc/NEWS.d/next/Library/2023-03-10-10-10-00.gh-issue-102573.GQ7sV6.rst

```diff
@@ -0,0 +1 @@
+Use ``set``-based name lookup rather than ``tuple``\s for frozen dataclasses
```

Comment:
This news entry only describes the changes you have made to the internal implementation, but users of frozen dataclasses generally don't care about that kind of thing. You should change the news entry so that it describes the impact that this change will have on users of frozen dataclasses.

E.g. does this speed up the time it takes to set certain attributes? To construct frozen dataclass instances? If it does speed certain things up, how much does it speed things up by?

Reply:
I update the news entry with an appropriate description.